docs/dev/adding_instruments.rst

```diff
@@ -187,92 +227,78 @@ In addition, any entries in ``**kwargs**`` take precedence, so if they need to,
+    >>> extreme.cell_temp  # Sends ":TEMP?" to the device
+    127.2
 
 The :func:`Instrument.control <pymeasure.instruments.Instrument.control>` function extends this behavior by creating a property that you can read and set. For example, if our "Extreme 5000" has the :code:`:VOLT?` and :code:`:VOLT <float>` commands that are in Volts, we can write the following property.
 
 .. testcode::
 
     Extreme5000.voltage = Instrument.control(
         ":VOLT?", ":VOLT %g",
-        """ A floating point property that controls the voltage
-        in Volts. This property can be set.
-        """
+        """Control the voltage in Volts (float)."""
     )
 
-You will notice that we use the `Python string format`_ :code:`%g` to pass through the floating point.
+You will notice that we use the `Python string format`_ :code:`%g` to format passed-through values as floating point.
 
 .. _Python string format: https://docs.python.org/3/library/string.html#format-specification-mini-language
 
-We can use this property to set the voltage to 100 mV, which will execute the command and then request the current voltage.
+We can use this property to set the voltage to 100 mV, which will send the appropriate command and then requests the current voltage.
 
 .. doctest::
 
     >>> extreme = Extreme5000("GPIB::1")
-    >>> extreme.voltage = 0.1        # Executes ":VOLT 0.1"
-    >>> extreme.voltage              # Reads ":VOLT?"
+    >>> extreme.voltage = 0.1        # Sends ":VOLT 0.1"
+    >>> extreme.voltage              # Sends ":VOLT?" to query for the current value
     0.1
 
-Using :func:`Instrument.control <pymeasure.instruments.Instrument.control>` and :func:`Instrument.measurement <pymeasure.instruments.Instrument.measurement>` functions, you can create a number of properties for basic measurements and controls. 
+Finally, the :func:`Instrument.setting <pymeasure.instruments.Instrument.setting>` function can only set, but not read values.
 
-The :func:`Instrument.control <pymeasure.instruments.Instrument.control>` function can be used with multiple values at once, passed as a tuple. Say, we may set voltages and frequencies in our "Extreme 5000", and the the commands for this are :code:`:VOLTFREQ?` and :code:`:VOLTFREQ <float>,<float>`, we could use the following property:
+Using the :func:`Instrument.control <pymeasure.instruments.Instrument.control>`, :func:`Instrument.measurement <pymeasure.instruments.Instrument.measurement>` and :func:`Instrument.control <pymeasure.instruments.Instrument.control>` functions, you can create a number of properties for basic measurements and controls.
 
-.. testcode::
+The next sections detail additional features of :func:`Instrument.control <pymeasure.instruments.Instrument.control>` that allow you to write properties that cover specific ranges, or have to map between a real value to one used in the command. Furthermore it is shown how to perform more complex processing of return values from your device.
```

Comment:
including the derivatives as well or does the user understand, that `measurement` and `setting` work the same?
```suggestion
The next sections detail additional features of :func:`Instrument.control <pymeasure.instruments.Instrument.control>` and derivatives that allow you to write properties that cover specific ranges, or have to map between a real value to one used in the command. Furthermore it is shown how to perform more complex processing of return values from your device.
```

Reply:
Noted, I instead used the term "property factories" from the previous heading.